redux/store.ts

```diff
@@ -1,9 +1,13 @@
 import { createStore, combineReducers, applyMiddleware } from "redux";
 
 import { todoSearch } from "./TodoSearch/reducer";
+import { authReducer } from "./auth/reducer";
+import { isSignupReducer } from "./signupState/reducer";
 
 const rootReducer = combineReducers({
   todoSearch,
+  isSignupReducer,
```

Comment:
this could be key: value for the readable purpose. 
Ex: isSignUp: isSignupReducer